src/set_theory/game.lean

```diff
@@ -168,3 +178,252 @@ def ordered_add_comm_group_game : ordered_add_comm_group game :=
   ..game_partial_order }
 
 end game
+
+namespace pgame
+
+/-! Multiplicative operations can be defined at the level of pre-games,
+but to prove their properties we need to use the abelian group structure of games.
+Hence we define them here. -/
```

Comment:
Again, I don't really understand the "hence" here. Why is needing the abelian group structure to prove properties a reason to move the definition of multiplication earlier?

Reply:
My understanding is that this comment explains why this definition is in `game.lean` as opposed to `pgame.lean`. It's not attempting to describe why it was moved (that's what the commit message is for), but why it is here and not in the file about pgames

Reply:
That's correct. I can switch the the wording to something like

"Hence we define the multiplication of pre-games in this file instead of pgame.lean"

Reply:
`game.lean` provides computationally useful tools for `pgame` proofs and is only ever used in `surreal.lean`. So the `game.lean` file becomes a single file to import for doing computations using `pgame`. 

The proofs about `mul` really have nothing to do with `surreal` or `numeric`. These theorems *almost* provide a ring structure on `game`. Even if there isn't a complete ring instance you get several computational `simp` lemmas involving `game`s like `⟦x * (y + z) ⟧ = ⟦x * y⟧ + ⟦x * z⟧`.

Reply:
Ah! Okay. I was reading this as an explanation of why this material was in `game` rather than `surreal`, but you're actually explaining why it is in `game` rather than `pgame`, and as such it makes much more sense.